Zend/zend_API.h

```diff
@@ -2306,6 +2318,15 @@ static zend_always_inline bool zend_parse_arg_str(zval *arg, zend_string **dest,
 	return zend_parse_arg_str_ex(arg, dest, check_null, arg_num, /* frameless */ false);
 }
 
+static zend_always_inline zend_string *zend_parse_arg_str_no_null(zval *arg, uint32_t arg_num)
+{
+	if (EXPECTED(Z_TYPE_P(arg) == IS_STRING)) {
+		return Z_STR_P(arg);
+	} else {
+		return zend_parse_arg_str_slow(arg, arg_num);
+	}
+}
+
```

Comment:
While I agree with getting rid of the dest param for the _slow functions, does this particular extra function actually change anything? Please check the asm; given that it's force inlined it ought to be equivalent to zend_parse_arg_str_ex.